### PR TITLE
[Documentation] mincpik perldoc

### PR DESCRIPTION
**Pull request by nicolasbrossard**

### CODE REVIEW

File: dicom-archive/updateMRI_Upload.pl [hunk 1]
```diff
@@ -3,6 +3,90 @@
 # zia.mohades@mcgill.ca
 # Perl tool to update the mri_upload table
 
+=pod
```
Comment (cmadjar) [inline dicom-archive/updateMRI_Upload.pl]: The changes in the updateMRI_Upload.pl file should not be present in the PR

File: docs/scripts_md/updateMRI_Upload.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# NAME
```
Comment (cmadjar) [inline docs/scripts_md/updateMRI_Upload.md]: This updateMRI_Upload.pl MD file should not be part of the PR

File: uploadNeuroDB/bin/mincpik [hunk 3]
```diff
@@ -1,4 +1,156 @@
 #! /usr/bin/env perl
+
```
Comment (cmadjar) [inline uploadNeuroDB/bin/mincpik]: Missing the MD file for mincpik

File: uploadNeuroDB/bin/mincpik [hunk 4]
```diff
@@ -1,4 +1,156 @@
...
+=pod
+
+=head1 NAME
+
+mincpik - generate an image from a set of MINC volumes
+
+=head1 SYNOPSIS
+
+mincpik [options] minc_file [outfile]
```
Comment (cmadjar) [inline uploadNeuroDB/bin/mincpik]: Maybe mention the the available [options] are listed in the description?

File: uploadNeuroDB/bin/mincpik [hunk 5]
```diff
@@ -1,4 +1,156 @@
...
+=item *
+B<-verbose> : be verbose
+
+=item *
+B<-clobber> : if it exists, overwrite the output file. An error is produced if F<outfile> exists
+and this option is not used.
+
+=item *
+B<-fake> : do not actually run the command but instead print on STDOUT all the calls to the
+ scripts and external tools (see above) issued to generate the image
```
Comment (cmadjar) [inline uploadNeuroDB/bin/mincpik]: misalignement of script in the terminal

File: uploadNeuroDB/bin/mincpik [hunk 6]
```diff
@@ -1,4 +1,156 @@
...
+integer representation. Default is the full range for the type and sign. 
+This option is ignored for floating point values. See manpage for C<mincreshape>.
+
+=item *
+B<-image_range min max> : Normalize the image to a given C<min> and C<max> real value (not
+voxel value). Cannot be used with C<-auto_range>. See manpage for C<mincreshape>.
+
+=item *
+B<-auto_range> : Automatically find the range used to normalize the image. This is
+done by finding the 5% to 95% PcT image range using C<mincstats>. This option
```
Comment (cmadjar) [inline uploadNeuroDB/bin/mincpik]: what is PcT?

File: uploadNeuroDB/bin/mincpik [hunk 7]
```diff
@@ -1,4 +1,156 @@
...
+
+None reported.
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+Andrew Janke - a.janke@gmail.com
```
Comment (cmadjar) [inline uploadNeuroDB/bin/mincpik]: Add a comma after the first author, so that when displayed in one row there is a separation.

File: uploadNeuroDB/bin/mincpik (hunk 4 above)
Comment (nicolasbrossard) [reply uploadNeuroDB/bin/mincpik]: I suggest to leave it like this (that's the way the synopsis is described in a Unix manual page)

File: uploadNeuroDB/bin/mincpik (hunk 6 above)
Comment (nicolasbrossard) [reply uploadNeuroDB/bin/mincpik]: Postponing documentation of this. it for now. Will have to ask Bert about this one...

File: uploadNeuroDB/bin/mincpik (hunk 7 above)
Comment (nicolasbrossard) [reply uploadNeuroDB/bin/mincpik]: Ok

File: uploadNeuroDB/bin/mincpik (hunk 3 above)
Comment (nicolasbrossard) [reply uploadNeuroDB/bin/mincpik]: Yikes! My bad...
